### ISSUE TITLE
`@"= \, -1"` parses but throws on display

### ISSUE THREAD

**Issue by charlesroddie** [verybadcat/CSharpMath#213]
`@"= \, -1"` parses (`CSharpMath.Atom.LaTeXParser.MathListFromLaTeX`) but throws on display.

```
CSharpMath.Structures.InvalidCodePathException: Invalid space between Relation and Binary Operator
```

**Comment by charlesroddie** [comment]
A code search leads to https://github.com/verybadcat/CSharpMath/blob/a7e11bae0e5f37d0de61d2442c72f270522de3cb/CSharpMath/Display/InterElementSpaces.cs but this code isn't intelligible to me.

**Comment by Happypig375** [comment]
Right. This was refactored a little bit but much of the logic is from iosMath which I never understood.

**Comment by Happypig375** [comment]
Fixed by #249 (merged as [`dbc9e30`](https://github.com/verybadcat/CSharpMath/commit/dbc9e30ebe031ee91bd2bd3ebb84682e6c3e81c3)), including regression coverage for the typesetting path. Closing as completed after the author-confirmation period elapsed without a contrary report.
